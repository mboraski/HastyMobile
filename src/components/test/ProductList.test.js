// TODO: Remove when tests implemented
test.only('real recognize real', () => {
    expect('real').toBe('real');
});
// import 'react-native';
// import React from 'react';
// import { shallow } from 'enzyme';
//
// import ProductList from '../ProductList';
//
// describe('ProductList', () => {
//     it('renders correctly', () => {
//         const cart = {
//             totalProducts: 0,
//             products: {}
//         };
//         const products = [
//             {
//                 quantity: 1,
//                 title: 'Type 1 code 1',
//                 thumbnail_image: 'image',
//                 price: '$3.49',
//                 productCode: '1'
//             }
//         ];
//         const wrapper = shallow(<ProductList cart={cart} products={products} />);
//         expect(wrapper).toMatchSnapshot();
//     });
//
//     it('renders correctly with selected item', () => {
//         const cart = {
//             totalProducts: 1,
//             products: {
//                 1: {
//                     1: {
//                         quantity: 1
//                     }
//                 }
//             }
//         };
//         const products = [
//             {
//                 quantity: 1,
//                 title: 'Type 1 code 1',
//                 thumbnail_image: 'image',
//                 price: '$3.49',
//                 productCode: '1'
//             }
//         ];
//         const wrapper = shallow(<ProductList cart={cart} products={products} />);
//         expect(wrapper).toMatchSnapshot();
//     });
// });
