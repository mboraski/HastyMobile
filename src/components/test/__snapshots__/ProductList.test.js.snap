// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductList renders correctly 1`] = `
<ScrollViewMock
  contentContainerStyle={
    Object {
      "alignItems": "flex-start",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "paddingLeft": 20,
      "paddingRight": 5,
    }
  }
>
  <ProductDetail
    key="1"
    onPress={[Function]}
    product={
      Object {
        "price": "$3.49",
        "productCode": "1",
        "quantity": 1,
        "thumbnail_image": "image",
        "title": "Type 1 code 1",
      }
    }
    style={
      Object {
        "marginBottom": 10,
        "marginRight": 8,
        "width": 139,
      }
    }
  />
</ScrollViewMock>
`;

exports[`ProductList renders correctly with selected item 1`] = `
<ScrollViewMock
  contentContainerStyle={
    Object {
      "alignItems": "flex-start",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "paddingLeft": 20,
      "paddingRight": 5,
    }
  }
>
  <ProductDetail
    key="1"
    onPress={[Function]}
    product={
      Object {
        "price": "$3.49",
        "productCode": "1",
        "quantity": 1,
        "thumbnail_image": "image",
        "title": "Type 1 code 1",
      }
    }
    style={
      Object {
        "marginBottom": 10,
        "marginRight": 8,
        "width": 139,
      }
    }
  />
</ScrollViewMock>
`;
